src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Address.java

```diff
@@ -94,8 +93,7 @@ public String getFullAddress() {
                             line1,
                             line2,
                             town,
-                            country,
-                            portOfEntry.getLocationCode())
+                            country)
```

Comment:
Add postcode before country. We should have 5 lines. We decided to keep it the same as UK addresses, except we drop county and put country last.